services/motion/builtin/move_request.go

```diff
@@ -518,8 +518,44 @@ func (ms *builtIn) newMoveOnGlobeRequest(
 		return nil, err
 	}
 
+	// convert obstacles of type []GeoGeometry into []Geometry
 	geomsRaw := spatialmath.GeoGeometriesToGeometries(obstacles, origin)
 
+	// convert bounding regions which are GeoGeometries into Geometries
+	boundingRegions := req.BoundingRegions
+	if boundingRegions == nil {
+		boundingRegions = []*spatialmath.GeoGeometry{}
+	}
+	interactionSpaces := spatialmath.GeoGeometriesToGeometries(boundingRegions, origin)
+
+	// Here we determine if the robot is contained by the defined interaction spaces.
+	// TODO(nf): Want to show if the robot is contained by the union of the defined interaction spaces
```

Comment:
Not sure if you wanted to commit this or not but lets remove for something more consise before we merge

Reply:
will remove in favor of a more concise statement.